content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -333,6 +333,10 @@ The nodes are where your workloads (containers and Pods, etc) run. To add new no
   kubeadm join --token <token> <control-plane-host>:<control-plane-port> --discovery-token-ca-cert-hash sha256:<hash>
   ```
 
+{{< note >}}
+On Windows you may need to specify the `cri-socket` parameter.  For example, with Containerd as the runtime add `--cri-socket "npipe:////./pipe/containerd-containerd"` to the `kubeadm join` command
```

Comment:
When do users not need to specify the `cri-socket` parameter for Windows?
If we say `may need to` I think we should give guidance on when they would or wouldn't need to specify the extra arg.

Reply:
Makes sense.  I believe it is required for any remote runtimes.  If using docker shim then it is not required.  Since these docs are for 1.24+ should I be more direct and be more direct that it is requird something like:

```suggestion
On Windows when using a remote runtime the `cri-socket` parameter is required.  For example when using Containerd as the runtime add `--cri-socket "npipe:////./pipe/containerd-containerd"` to the `kubeadm join` command
```

Reply:
I think from Tuesday (the expected v1.24 release date), setting a CRI socket path becomes mandatory.

Reply:
Is this required then?

Reply:
Maybe we should say:

On Windows you should specify the `cri-socket` parameter in the `kubeadm join` command. The default path is "npipe:////./pipe/containerd-containerd" for containerd.

Reply:
updated